draft-ietf-quic-http.md

```diff
@@ -159,7 +159,8 @@ multiplexing, and flow control.  Discovery of an HTTP/3 endpoint is described in
 Within each stream, the basic unit of HTTP/3 communication is a frame
 ({{frames}}).  Each frame type serves a different purpose.  For example, HEADERS
 and DATA frames form the basis of HTTP requests and responses
-({{request-response}}).
+({{request-response}}).  Frames that apply to the entire connection are
+conveyed on a dedicated control stream.
```

Comment:
This subtly hints that control messages are well ordered across each other, in contrast to native QUIC control messages.  I got a bit worried about `MAX_PUSH_ID` on my initial read-through as a result, and knowing about the existence of the control stream earlier would have helped.

Reply:
The subtlety of the hint is appropriate.  We don't necessarily need ordering for something (MAX_PUSH_ID included), but we benefit from it for some uses.  We don't need to be clear about what has ordering requirements and what doesn't, but this is helpful.

Reply:
It also (IIUC) doesn't preclude use of non-STREAM QUIC frames for future HTTP work (datagram?).